arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -669,19 +795,29 @@
     },
     "StorageAccount": {
       "description": "The properties of a storage account associated with this resource.",
+       "example": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+        "isPrimary": true
+      },
       "properties": {
         "id": {
-          "description": "The id of the storage account resource.",
+          "description": "The id of the storage account resource. Media Services relies on tables and queues as well as blobs, so the primary storage account must be a Standard Storage account (either Microsoft.ClassicStorage or Microsoft.Storage). Blob only storage accounts can be added as secondary storage accounts (isPrimary false).",
           "type": "string"
         },
         "isPrimary": {
-          "description": "Is this storage account resource the primary storage account for the Media Service resource.",
+          "description": "Is this storage account resource the primary storage account for the Media Service resource. Blob only storage must set this to false.",

```

Comment:
Seems like it's missing something. Ie: "Indicates whether this storage account ....", or perhaps a question: "Is this storage account resource the primary storage account for the Media Service resource?"